CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
 # App Center SDK for Unity Change Log
 
+## Version 4.2.0
+
+### AppCenter
+
+#### UWP
+
+* **[Feature]** In the App Center SDK was added support of strong-named signing assemblies for compatibility with applications that use strong-named signing. Read more about [strong-named assemblies](https://docs.microsoft.com/en-us/dotnet/standard/assembly/strong-named).
+* **[Fix]** Fix SDK not sending events when application is packed to self-contained single-file executable.
```

Comment:
Not related

Reply:
So this time the version number should be 4.1.1?

Reply:
If it's only fixes, so yes